Code/GraphMol/FileParsers/testExtendedStereoParsing.cpp

```diff
@@ -0,0 +1,103 @@
+//
+//  Copyright (C) 2017 Greg Landrum
```

Comment:
copyright here and on other new files could be corrected (it doesn't need to be mine)

Reply:
I was thinking 2018 and just RDK

Reply:
There’s no legal entity for the RDKit; so that unfortunately won’t work.